### PR TITLE
rest repo added

### PR DESCRIPTION
**Pull request by CPernet**
bids_validator --bep006 happy

### CODE REVIEW

File: eeg_rest_fmri/task-rest_eeg.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+"TaskName": "rest",
+"TaskDescription": "Subjects were asked to remain still and their head was immobilised using a vacuum cushion during scanning",
+"SamplingFrequency": 5000,
+"PowerLineFrequency": 50,
+"ManufacturersModelName": "BrainAmp",
```
Comment (sappelhoff) [inline eeg_rest_fmri/task-rest_eeg.json]: probably `ManufacturersModelName: BrainAmp MR`... right?

File: eeg_rest_fmri/task-rest_bold.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "TaskName": "rest",
```
Comment (sappelhoff) [inline eeg_rest_fmri/task-rest_bold.json]: missing indentation

File: eeg_rest_fmri/task-rest_eeg.json (hunk 1 above)
Comment (CPernet) [reply eeg_rest_fmri/task-rest_eeg.json]: probably but Jon isn't sure

File: eeg_rest_fmri/task-rest_bold.json (hunk 2 above)
Comment (CPernet) [reply eeg_rest_fmri/task-rest_bold.json]: done the edit and pushed

**Comment by sappelhoff** [comment]
excellent, thanks @CPernet !

**Pull request merged**